lib/tasks/ingest.rake

```diff
@@ -389,6 +389,44 @@ namespace :ingest do
     end
   end
 
+  desc "Import all NYC schools"
+  # task import_all_nyc_schools: :environment do
+  task import_all_nyc_schools: :environment do
+    csv_text = File.read('data/2016_-_2017_School_Locations.csv')
+    rows = CSV.parse(csv_text, headers: true)
+    rows.each do |row|
+      row_hash = row.to_hash
+      school = School.where(code: row_hash['ATS SYSTEM CODE'].strip).first_or_initialize
+      school.name = school.name || row_hash['LOCATION_NAME'].strip
+      school.address_line_1 = school.address_line_1 || row_hash['PRIMARY_ADDRESS_LINE_1'].strip
+      school.state = school.state || row_hash['STATE_CODE'].strip
+      if row_hash['Location 1'].present?
+        school.address_line_2 = school.address_line_2 || row_hash['Location 1'].strip
+        school.borough = school.borough || borough(row_hash['Location 1'].strip)
```

Comment:
Could we capitalize and trim the borough field?

Reply:
In addition, why is Location 1 lower-case, while other hash key's are upper-case? Can we keep consistency and have everything upper-case?

Reply:
@nnat425 the file comes this way from DOE.

@daryanypl we set the borough using a separate method which checks for the presence of the borough name in the string.  I added a line to raise an exception if the borough is not found.